frontend/sass/styles.scss

```diff
@@ -18,107 +21,41 @@
 @import "_tables";
 @import "_log-tools";
 @import "_repo-link";
-@import "_expandableArea";
 @import "_flexbox";
 @import "_examples";
+@import "side-nav";
 
 html,
 body,
 main,
 .usa-grid.site,
-#js-app, div[data-reactroot=""] {
-  height: 100%; }
-
-.recent-drafts {
-  width: 100%;
-}
-
-.pull-right {
-  float: right;
-}
-
-.pull-right::after {
-  content: '';
-  clear: both;
-  display: table-cell;
+#js-app,
+div[data-reactroot=''] {
+  height: 100%;
 }
 
-.list-group {
+.recent-drafts {
```

Comment:
Looks like this can be removed also. I don't see it used anywhere, and the name makes me think it is a relic of when Federalist had a content editor.